ubuntu/rsyslog-logzio/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && apt-get -y -q install software-properties-common python-software-properties
 RUN add-apt-repository ppa:adiscon/v8-stable
-RUN apt-get update && apt-get -y -q install rsyslog=8.23.0-0adiscon1trusty1
```

Comment:
docker build broke with this pinned down.  I forget why I did this in the first place.